it/pages/TwoEyeReviewerResultControllerISpec.scala

```diff
@@ -32,6 +32,7 @@ class TwoEyeReviewerResultControllerISpec extends IntegrationSpec {
 
```

Comment:
We need an equivalent integration test for the fact check confirmation controller.